src/scanner.rs

```diff
@@ -410,8 +420,8 @@ where
                 syn::TraitItem::Macro(m) => {
                     self.data.skipped_macros.add(m);
                 }
-                syn::TraitItem::Verbatim(v) => {
-                    self.syn_info("skipping Verbatim expression", v);
+                syn::TraitItem::Verbatim(_v) => {
+                    // self.syn_info("skipping Verbatim expression", v);
```

Comment:
hmm, is this generating a bunch of extraneous `INFO` level logs? we could just remove if it's not being helpful